### DISCUSSION TITLE
Why not using GitHub releases instead of putting img's and iso's in the source tree?

### DISCUSSION THREAD

**Discussion by dalmemail** [VendelinSlezak/BleskOS#81] [Ideas]
I think the title says it all. See https://github.com/rockdaboot/wget2/ and https://github.com/rockdaboot/wget2/releases for an example.

That way you could put both the changes and the img & iso files there, and use the source tree for the source and the README for holding just the most information for someone new staring at your project, instead of the changelog.

You don't need to publish a new release everytime you do changes, just when huge changes are added or a lot of time has happened from the last release. Your compile.sh scripts works well and fast so it's not hard for anyone to compile the system themselves.

Just my 2 cents. Btw, cool project!

**Comment by VendelinSlezak** [comment]
That sounds like good idea. I still have to learn many things about how to use GitHub properly :) Anyway I will be still out from programming BleskOS for some time from now, but when I will come back, I will surely reorganize images to releases.

**Comment by VendelinSlezak** [comment]
Update 31 is published as release.

**Reply by dalmemail** [reply]
Great! BTW, where is the project heading to? Are you planning to implement a libc / posix?

**Reply by VendelinSlezak** [reply]
Basically said, I am heading to generally improve everything what is now in BleskOS and I also want to add many new features. I have tons of ideas in head for whose I only need some time to write them, just to mention few of them, implementing drawing of normal scalable fonts, support for ICO files, feature for compriming bunch of JPG files to desired size range at one click, showing images in internet browser, Spreadsheet editor with support for XLSX and ODS and more.

Currently it is not my goal to fully implement libc or to be posix compatible, as BleskOS is now intended only for running programs written directly for it.
